packages/liferay-jest-junit-reporter/index.js

```diff
@@ -2,41 +2,95 @@
 
 const xml = require('xml');
 const fs = require('fs');
+const path = require('path');
+const stripAnsi = require('strip-ansi');
 
-module.exports = (report) => {
-  const generalMetrics = {
-    _attr: {
-      hostname: '',
-      id: 0,
-      errors: 0,
-      failures: report.numFailedTests,
-      name: 'Jest',
-      package: '',
-      skipped: 0,
-      tests: report.numTotalTests,
-      time: 0,
-      timestamp: report.startTime
-    }
-  };
-
-  let testResults = report.testResults
-    .reduce((results, suite) => suite.testResults.length ? results.concat(suite.testResults) : results, [])
-    .map(testCase => {
-      return {
-        'testcase': [{
-          _attr: {
-            classname: testCase.ancestorTitles[0],
-            name: `${testCase.ancestorTitles.join(' ')} ${testCase.title}`,
-            time: testCase.duration / 1000
-          }
-        }]
-      };
-    });
-
-  fs.writeFileSync('TEST-frontend-js.xml', xml(
-    {'testsuite': [generalMetrics, ...testResults]},
-    { declaration: true, indent: '  '}
-  ));
-
-  return report;
+const NEW_LINE = '\n';
+
+const RELATIVE_APPS_DIR = '/modules/apps/';
```

Comment:
I think this should be fine for now, since this is mostly to be consumed by Portal modules. To ease your concerns a little bit... maybe we could make this a configuration option, being `/modules/apps/` the default? That way we protect for future usage if necessary?